### DISCUSSION TITLE
How to remove custome domain from github pages ?

### DISCUSSION THREAD

**Discussion by anuragjain-git** [community/community#84735] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

- I removed my custom domain from ```settings -> pages```, but my repositories still redirects to my customdomain.
- I tried unpublishing and republishing my repositories, it didn't work.

### -> settings -> pages (here i removed my verified domain)

![image](https://github.com/community/community/assets/98457054/607ccb5f-ce2f-4299-9c93-ba57ecc1544b)

### After publishing a repository in ```repo -> settings -> pages``` :
![image](https://github.com/community/community/assets/98457054/2aff2d2f-9bcd-407e-98af-cf42cdab7a30)

**Comment by fury-05** [comment, marked as answer]
You can do as follows [Link](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#removing-a-custom-domain)

But assuming you have already done this.

Delete the dns record from your domain manager.

Clear browser cache and wait for some time as dns propagation takes some time.

**Reply by anuragjain-git** [reply]
Thanks btw :

1. I removed the cname record from DNS Management.
2. Then i ``Remove`` my custom domain in ```https://github.com/anuragjain-git/anuragjain-git.github.io/settings/pages``` under Custom domain.
3. And then unpublished and republished my repositories and it works.

**Comment by cosmicqbit** [comment]
If you want to publish your site at username.github.io/site create a repository under the name username.github.io and push your site directory to the repo. You should be able to access your site at username.github.io/site address.

Like for example [this repo](https://github.com/cosmicqbit/js-projects) of mine It was previously named cosmicqbit.github.io instead of js-projects and all the directories (simple web dev projects) uploaded in it were accessible at cosmicqbit.github.io/project-name.
